.github/workflows/ci.yml

```diff
@@ -56,34 +54,28 @@ jobs:
         with:
           prefix-key: "0" # change this to invalidate CI cache
           shared-key: "cargo_nextest-${{ matrix.cache_id }}"
-      - run: cargo llvm-cov show-env | tr -d "'" >> "$GITHUB_ENV"
 
       # Run unit tests
-      - run: cargo nextest run --locked --workspace --exclude integration-tests --cargo-profile dev-release --profile ci ${{ matrix.flags }}
+      - run: just codecov "nextest-unit ${{ matrix.type }}"
         env:
           RUST_BACKTRACE: short
-      - run: cargo llvm-cov report --profile dev-release --codecov --output-path unittests.json
       - uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          files: unittests.json
+          files: codecov.json
           fail_ci_if_error: true
           flags: unittests,${{ matrix.id }}
       # See https://github.com/taiki-e/cargo-llvm-cov/issues/292
       - run: find target -name '*.profraw' -delete
 
       # Run integration tests
-      - run: cargo nextest run --locked --package integration-tests --cargo-profile dev-release --profile ci ${{ matrix.flags }}
-        if: matrix.run_integ_tests
+      - run: just codecov "nextest-integration ${{ matrix.type }}"
         env:
           RUST_BACKTRACE: short
```

Comment:
Observation, with `env_var_or_default` we could move this into the Justfile with something along the lines of

```just
export RUST_BACKTRACE := get_env_or_default("RUST_BACKTRACE", "short")
```

or

```just
rust_backtrace := get_env_or_default("RUST_BACKTRACE", "short")

nextest_integration: $RUST_BACKTRACE="{{rust_backtrace}}"
    ...
```

Reply:
Hmm so I was actually wondering, what’s the point of setting the backtrace to `short` in CI? I remember digging through CI and not finding enough details to pinpoint the failure, so I’m thinking CI should have at least a full backtrace, though local development could also work with short backtraces.

So I’m curious, should we not just make it a `RUST_BACKTRACE=1` alongside moving it to the Justfile?

Reply:
My opinion is that the fluff that `short` filters out is very, very rarely useful in programs that don’t deal with some particularly low level details (which neard does not.)